test/fixtures/input/footnote-trailing-whitespace.text

```diff
@@ -0,0 +1 @@
+[^1]: Footnote with trailing newline and space\n 
```

Comment:
This isn’t a line feed, but a slash followed by the letter `n`! Could you add an actual line feed?

In `remark-lint` we have a [character mapping](https://github.com/remarkjs/remark-lint/blob/master/script/characters.js) to make it obvious that a trailing space is used, but that isn’t available here. Might be an idea for the future.